web3_proxy/src/referral_code.rs

```diff
@@ -22,11 +22,13 @@ impl TryFrom<String> for ReferralCode {
     type Error = anyhow::Error;
 
     fn try_from(x: String) -> Result<Self> {
-        if !x.starts_with("llamanodes-") {
-            return Err(anyhow::anyhow!(
-                "Referral Code does not have the right format"
-            ));
-        }
+        // TODO: Check if string is a valid ULID
```

Comment:
i don't think we actually want this todo. it can be useful to make "vanity" referral codes that won't have the default ULID structure.